src/surge-xt/gui/overlays/Oscilloscope.cpp

```diff
@@ -467,21 +490,27 @@ void SpectrumDisplay::resized()
               // We're essentially filling with the calculated equivalent of the bottom of the
               // graph, so the fact that we're not calling with the actual noise floor or dB ceiling
               // is immaterial.
-              dbToY(-100.f, height, -100.f, 0.f));
+              dbToY(-96.f, height, -96.f, 0.f));
 }
 
 void SpectrumDisplay::updateScopeData(internal::FftScopeType::iterator begin,
                                       internal::FftScopeType::iterator end)
 {
     // Data comes in as gain.
     std::lock_guard l(data_lock_);
+
     // Decay existing data, and move new data in if it's larger.
-    const float decay = (1.f - params_.decay_rate);
+    const float decay = (1.f - (params_.decay_rate * 0.9f));
```

Comment:
With this change we no longer can get the instant-decay behavior when decay_rate = 1 or maxes behavior when decay_rate = 0.

Reply:
OK I will then just adjust the scaling to be sqrt of params_.decay_rate.